omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -663,8 +663,7 @@
 default['private_chef']['postgresql']['enable'] = true
 default['private_chef']['postgresql']['external'] = false
 default['private_chef']['postgresql']['ha'] = false
-default['private_chef']['postgresql']['dir'] = "/var/opt/opscode/postgresql/#{node['private_chef']['postgresql']['version']}"
-default['private_chef']['postgresql']['data_dir'] = "/var/opt/opscode/postgresql/#{node['private_chef']['postgresql']['version']}/data"
```

Comment:
I think this needs to be left in. This dir is an important attribute for postgres and gets used on multiple occassions.

Reply:
It was accidentally removed.  I already added these back in yesterday, and pushed :)